qiling/os/linux/function_hook.py

```diff
@@ -207,6 +207,11 @@ def get_ret_pc(self):
         # X8664
         elif  self.ql.archtype== QL_ARCH.X8664:
             return self.ql.unpack(self.ql.mem.read(self.ql.reg.rsp, self.ql.pointersize))
+
+        # SPARC
+        elif  self.ql.archtype== QL_ARCH.SPARC:
```

Comment:
x64 and sparc is the same ?

Reply:
sparc also has a return address pushed on the stack on call like x86 if that's what you mean